modules/text_on_image/text_on_image_59_by_PraneethaPradeepSandur.html

```diff
@@ -0,0 +1,97 @@
+<link href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap" rel="stylesheet">
```

Comment:
We generally score higher PRs that do not make external request. Learn from here and avoid links in your next PR. Notwithstanding, Well done.

Reply:
Sure. Thank you!